packages/bbot/changelog.yml

```
@@ -1,4 +1,20 @@
 # newer versions go on top
+- version: "1.0.2"
+  changes:
+    - description: Add integration screenshots
+      type: enhancement
+      link: https://github.com/elastic/integrations/pull/11742
+- version: "1.0.1"
+  changes:
+    - description: Tests use BBOT v2.1.2 NDJSON output
+      type: enhancement
+      link: https://github.com/elastic/integrations/pull/11742
+    - description: Update to support BBOT v2.0.0 event format
+      type: enhancement
+      link: https://github.com/elastic/integrations/pull/11742
+    - description: Add http_endpoint input option to support BBOT HTTP output module
+      type: enhancement
+      link: https://github.com/elastic/integrations/pull/11742
```

```suggestion
- version: "1.1.0"
  changes:
    - description: Update to support BBOT v2.0.0 event format.
      type: enhancement
      link: https://github.com/elastic/integrations/pull/11742
    - description: Add http_endpoint input option to support BBOT HTTP output module.
      type: enhancement
      link: https://github.com/elastic/integrations/pull/11742
```
- I don't think we need to mention the addition of screenshots.
- The changelog is user-facing, so we don't need to mention the testing changes.
- This is an enhancement, so bump minor (semver semantics).

---

OK, adjusting